core/modules/wpublish/QueriesAndChunks.cc

```diff
@@ -229,6 +231,7 @@ QueryStatistics::Ptr QueriesAndChunks::getStats(QueryId const& qId) const {
 void QueriesAndChunks::examineAll() {
     // Need to know how long it takes to complete tasks on each table
     // in each chunk, and their percentage total of the whole.
+    std::cout << "&&& QueriesAndChunks::examineAll a\n";
```

Comment:
I removed everything with "&&&" and did not push it to master after I committed. Basically, I just removed debugging instrumentation. The cout was a temporary workaround for the logger not working on a unit test. I just pushed the correct version to master.